device/android/src/main/java/com/capacitorjs/plugins/device/Device.java

```diff
@@ -78,4 +81,22 @@ public String getName() {
 
         return null;
     }
+
+    public String getWebViewVersion() {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            PackageInfo info = WebView.getCurrentWebViewPackage();
+            return info.versionName;
+        }
```

Comment:
```suggestion
        PackageInfo info = null;
        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
            info = WebView.getCurrentWebViewPackage();
        } else {
            PackageManager pm = this.context.getPackageManager();
            try {
                info = pm.getPackageInfo("com.google.android.webview", 0);
            } catch (PackageManager.NameNotFoundException e) {}
        }
        if (info != null) {
            return info.versionName;
        }
```

With this code you can also get the version on Android 5-6 devices, so only returns `Build.VERSION.RELEASE`, but ideally we should find a way of getting it on Android 7 too, because `Build.VERSION.RELEASE` is not helpful